packages/frontend/src/components/ResultsTable/ResultsTable.tsx

```diff
@@ -313,37 +314,46 @@ export const ResultsTable: FC<Props> = ({
                                                                             : [],
                                                                     )}
                                                                 >
-                                                                    <Draggable
-                                                                        key={
-                                                                            column.id
-                                                                        }
-                                                                        draggableId={
-                                                                            column.id
-                                                                        }
-                                                                        index={
-                                                                            index
+                                                                    <Tooltip2
+                                                                        content={
+                                                                            column.description ||
+                                                                            friendlyName(
+                                                                                column.id,
```

Comment:
@ZeRego  I left this here, as the table calculation column doesn't container either a `description` nor a `tableLabel` or `label`, so in this case their tooltip will be generated from their id 
<img width="191" alt="Screenshot 2022-01-20 at 13 10 45" src="https://user-images.githubusercontent.com/31137824/150396927-2554a7e5-83c0-4617-8849-cae03468ab5f.png">

Reply:
I would prefer if we set the table calculation in the useColumns hook and avoid handling a fallback here.
```
                            Header: (
                                <b>{friendlyName(tableCalculation.name)}</b>
                            ),
                            description: friendlyName(tableCalculation.name),
```